notebook/converted/imputation_viz.py

```diff
@@ -123,3 +123,88 @@
 plot += ggtitle('The relationship between sample count and val set loss')
 print(plot)
 
+
+# ## Evaluate Transfering Models
+
+# In[12]:
+
+
+in_files = glob.glob('../../results/transfer.*')
+print(in_files[:5])
+
+
+# In[17]:
+
+
+metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    
+    model_info = path.strip('.tsv').split('results/impute.')[-1]
+    model_info = model_info.split('.')        
+
+    supervised_model = model_info[4]
+    seed = model_info[6]
+    disease = model_info[3]
+             
+    new_df['supervised'] = supervised_model
+    new_df['seed'] = seed
+    new_df['disease'] = disease
+        
+    metrics = pd.concat([metrics, new_df])
+    
+metrics = metrics.reset_index()
+metrics['trial'] = metrics.index // 20
+metrics
+
+
+# In[18]:
+
+
+sepsis_df = metrics[metrics['disease'] == 'sepsis']
+tb_df = metrics[metrics['disease'] == 'tb']
+
+
+# In[19]:
+
+
+ggplot(sepsis_df, aes(x='train sample count', y='balanced_accuracy', color='factor(trial)')) + geom_line()
+
+
+# In[20]:
+
+
+plot = ggplot(sepsis_df, aes(x='train sample count', y='balanced_accuracy', color='factor(impute_samples)')) 
```

Comment:
Similar comment here, would be good to know exactly what the different factors/colors are.

Reply:
Makes sense, I added a note before the first visualization in the notebook since I use the term "trial" throughout without explaining it.